crowdloans/rewards/polkadot.json

```diff
@@ -1,4 +1,83 @@
 {
+  "2019": {
+    "cDot": "${amount * 1}",
+    "reward": {
+      "total": "5% of Total Supply",
```

Comment:
in the document its is 12% of total supply

Reply:
@Helen8 can you confirm please

Reply:
2019 - Composable Finance - 12% of supply
2031 - Centrifuge - 15% of supply

https://www.notion.so/parallelfinance/Batch-2-DOT-Crowdloan-345f2f01d71643febd8b3c6c935615eb